packages/next/pages/_error.tsx

```diff
@@ -65,6 +68,11 @@ export default class Error<P = {}> extends React.Component<P & ErrorProps> {
             </h2>
           </div>
         </div>
+        {process.env.__NEXT_RSC && (
+          <script>
+            {`(self.__next_s=self.__next_s||[]).push(new Error(${statusCode}))`}
```

Comment:
Can we use something like `self.__NEXT_DATA__.err` or `self.__NEXT_RSC_ERROR__`? I think `__next_s` should only contain serialized flight data.